scopes/react/react/react.env.ts

```diff
@@ -174,11 +168,8 @@ export class ReactEnv implements Environment {
   getDevServer(context: DevServerContext, targetConfig?: Configuration): DevServer {
     const defaultConfig = this.getWebpackConfig(context);
     const config = targetConfig ? webpackMerge(targetConfig as any, defaultConfig as any) : defaultConfig;
```

Comment:
fun fact - docs file is already included using the preview link file, as "defaultModule" at **generate-link.ts**

Reply:
tested in workspace, dev mode, and in bare scope 🐻